cumulusci/tasks/push/push_api.py

```diff
@@ -265,12 +268,24 @@ def __init__(self, sf, logger, lazy=None, default_where=None, batch_size=None):
             batch_size = 200
         self.batch_size = batch_size
 
-    def return_query_records(self, query):
-        res = self.sf.query_all(query)
-        if res["totalSize"] > 0:
-            return res["records"]
+        self.bulk = bulk
+
+    def return_query_records(self, query, field_names=None, sObject=None):
```

Comment:
sObject is the correct capitalization in prose, but I would stick with all lowercase `sobject` for a Python variable name. (Same thing below)